packages/Utils/test/CMakeLists.txt

```diff
@@ -1,5 +1,13 @@
 
 TRIBITS_ADD_EXECUTABLE_AND_TEST(
+  Miscellaneous_test
+  SOURCES tstMiscellaneous.cpp ${TEUCHOS_STD_UNIT_TEST_MAIN}
+  COMM serial mpi
```

Comment:
> The test is not enabled if I remove all together the COMM argument 

Let me take this back.  It is enabled but it runs with `mpiexec -np 4` although it could/should be run in serial.